README.adoc

```diff
@@ -75,88 +61,156 @@ It could be omitted as the title already says that it is an experimental feature
 * Avoid the phrase “It is recommended to” when proposing a solution.
 * Capitalize all Cypher keywords, i.e., `MATCH` rather than `match` or `Match`.
 
-=== Code
 
-All Neo4j codes follow the syntax `Neo.[Class].[Subclass].[Name]`.
-The notification codes’ [Class] is `ClientNotification`.
-Therefore, they all start with `Neo.ClientNotification`.
-The [Subclass] is one of `Statement`, `Procedure`, `Schema`, `Database`, `Security`, or `Request`.
-Code’s [Name] should be specific and explain what this code refers to.
+=== Status code elements
```

Comment:
Here we use the word 'elements', but in the text below and later we use 'components'